services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/document/EnhancedDocument.java

```diff
@@ -256,15 +281,15 @@ static Builder builder() {
     String getJson(String attributeName);
 
     /**
-     * Gets the {@link Boolean} value for the specified attribute.
+     * Gets the boolean value for the specified attribute.
      *
      * @param attributeName Name of the attribute.
      * @return value of the specified attribute in the current document as a non-null Boolean.
      * @throws RuntimeException
      *             if either the attribute doesn't exist or if the attribute
      *             value cannot be converted into a boolean value.
      */
-    Boolean getBoolean(String attributeName);
+    boolean getBoolean(String attributeName);
```

Comment:
We may want to keep it Boolean to differentiate between not existing and false

Reply:
Currenly even for v1 we donot have this behaviour https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-dynamodb/src/main/java/com/amazonaws/services/dynamodbv2/document/Item.java#L960-L987
We can add in future if there is a valid use case , for now user can use isPresent() check to see if the attribute exist

Reply:
Changed after internal discussion ,to Boolean and return null if value not present or Null